contracts/rewards/src/contract/query.rs

```diff
@@ -0,0 +1,186 @@
+use cosmwasm_std::Storage;
+use error_stack::Result;
+
+use crate::{
+    error::ContractError,
+    msg::RewardsPoolResponse,
+    state::{self, Epoch, PoolId},
+};
+
+pub fn get_rewards_pool(
```

Comment:
```suggestion
pub fn rewards_pool(
```
rust convention is to just use the name of the thing for getters